ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -139,10 +139,33 @@ def ocs_install_verification(
     # noobaa
     assert pod.wait_for_resource(
         condition=constants.STATUS_RUNNING,
-        selector=constants.NOOBAA_APP_LABEL,
-        resource_count=2,
+        selector=constants.NOOBAA_DB_LABEL,
+        resource_count=1,
+        timeout=timeout
+    )
+    assert pod.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector=constants.NOOBAA_OPERATOR_POD_LABEL,
+        resource_count=1,
         timeout=timeout
     )
+    assert pod.wait_for_resource(
+        condition=constants.STATUS_RUNNING,
+        selector=constants.NOOBAA_CORE_POD_LABEL,
+        resource_count=1,
+        timeout=timeout
+    )
+    # check noobaa CR for min number of noobaa endpoint pods
+    nb_obj = OCP(kind='noobaa', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    min_eps = nb_obj.get().get('items')[0].get('spec').get('endpoints').get('minCount')
```

Comment:
What about to check also max number.
`max_eps = nb_obj.get().get('items')[0].get('spec').get('endpoints').get('maxCount')`

Reply:
I see, my point was also cause of upgrade as we are running this verification after upgrade as well, so just thought about the case if it can happen that after upgrade we will have for example 3 pods but max will be set to 2.  But if you think it's not a problem here I am OK to not cover this scenario here.

Reply:
This is a very good point! Changed accordingly